jobs/dd-agent-windows/templates/bin/drain.ps1

```diff
@@ -0,0 +1 @@
+Start-Process -NoNewWindow -Wait "\var\vcap\packages\dd-agent-windows\Datadog\bin\agent.exe" -ArgumentList "stopservice"
```

Comment:
Should we do this part about printing an int to the output ? https://bosh.io/docs/drain/#script-implementation